presto-docs/src/main/sphinx/sql/create-schema.rst

```diff
@@ -40,6 +44,18 @@ Create the schema ``traffic`` if it does not already exist::
 
     CREATE SCHEMA IF NOT EXISTS traffic
 
+Create a new schema ``web`` and set the owner to user ``alice``::
+
+    CREATE SCHEMA web AUTHORIZATION alice
+
+Create a new schema ``web``, set the ``format`` property to ``ORC`` and set the owner to user ``alice``::
+
+    CREATE SCHEMA web AUTHORIZATION alice WITH ( format = 'ORC' )
```

Comment:
there is no such property for hive schemas. Hive only supports only `location` property so far.

Reply:
Oh right. That's for tables only.